src/frontend/parser.messages

```diff
@@ -4749,14 +4749,14 @@ Expected "in" after loop identifier.
 
 program: TRANSFORMEDDATABLOCK LBRACE FOR LPAREN WHILE
 ##
-## Ends in an error in state: 354.
+## Ends in an error in state: 355.
 ##
-## nested_statement -> FOR LPAREN . identifier IN lhs COLON lhs RPAREN statement [ WHILE VOID VECTOR UPPER UNITVECTOR TRUNCATE TARGET SIMPLEX SEMICOLON ROWVECTOR RETURN REJECT REALNUMERAL REAL RBRACE PROFILE PRINT POSITIVEORDERED PLUS ORDERED OFFSET MULTIPLIER MINUS MATRIX LPAREN LOWER LBRACK LBRACE INTNUMERAL INT INCREMENTLOGPROB IF IDENTIFIER GETLP FOR ELSE COVMATRIX CORRMATRIX CONTINUE COMPLEX CHOLESKYFACTORCOV CHOLESKYFACTORCORR BREAK BANG ARRAY ]
-## nested_statement -> FOR LPAREN . identifier IN lhs COLON non_lhs RPAREN statement [ WHILE VOID VECTOR UPPER UNITVECTOR TRUNCATE TARGET SIMPLEX SEMICOLON ROWVECTOR RETURN REJECT REALNUMERAL REAL RBRACE PROFILE PRINT POSITIVEORDERED PLUS ORDERED OFFSET MULTIPLIER MINUS MATRIX LPAREN LOWER LBRACK LBRACE INTNUMERAL INT INCREMENTLOGPROB IF IDENTIFIER GETLP FOR ELSE COVMATRIX CORRMATRIX CONTINUE COMPLEX CHOLESKYFACTORCOV CHOLESKYFACTORCORR BREAK BANG ARRAY ]
```

Comment:
Why is IMAGNUMERIAL showing up here now?

Reply:
Why wouldn't it? There's already INTNUMERAL and REALNUMERAL. Or are you asking why didn't it show up back in #964 ? The reason seems to be that that pull didn't update `parser.messages` at all. (the reviewer probably should have asked about that...)

Reply:
Our tests make sure that the parser messages are up to date in terms of each sentence leads to an error and all error states are represented, but it doesn't update the comments automatically. I ran `dune build @update_messages` as part of this PR

Reply:
Right, adding IMAGNUMERAL did not create new parser states so update was not necessary.

Reply:
icic cool cool just checking